src/app/core/core.env.ts

```diff
@@ -7,6 +7,7 @@ const DEFAULTS = {
   CMS_TTL: 1, // 1 second
   CMS_ROOT_TTL: 3600, // 1 hour
   CMS_USE_LOCALSTORAGE: false,
+  FEEDER_HOST: 'https://feeder.prx.org',
```

Comment:
Not sure. I needed something for the HalRemote, but honestly at the time I wrote this I had no idea we'd have a full URL as a link on the story distro.